tests/CrawlerProcess/asyncio_deferred_signal.py

```diff
@@ -0,0 +1,45 @@
+import asyncio
+import sys
+
+from scrapy import Spider
+from scrapy.crawler import CrawlerProcess
+from scrapy.utils.defer import deferred_from_coro
+from twisted.internet.defer import Deferred
+
+
+class UppercasePipeline:
+    async def _open_spider(self, spider):
+        spider.logger.info("async pipeline opened!")
+        await asyncio.sleep(0.1)
+
+    def open_spider(self, spider):
+        loop = asyncio.get_event_loop()
```

Comment:
is this line needed now?

Reply:
Good eye, it is not. I wonder why the flake8 check didn't fail 🤔